src/views/tenant/administration/EditTenant.js

```diff
@@ -32,17 +32,22 @@ const EditTenant = () => {
   useEffect(() => {
     if (!tenantDomain) {
       ModalService.open({
-        body: 'Error: Invalid request. Could not load requested tenant.',
+        body: 'Error: INvalid request. Could not load requested tenant.',
         title: 'Invalid Request',
       })
       setQueryError(true)
     }
   }, [tenantDomain, dispatch])
 
   const onSubmit = (values) => {
-    // @todo bind this
-    //window.alert(JSON.stringify(values))
-    genericPostRequest({ path: '/api/EditTenant', values })
+    const shippedValues = {
+      Tenant: tenantDomain,
```

Comment:
Your API is expecting tenantid: 

```powershell
$Tenant = $request.body.tenantid
```

Reply:
This is why we have a consistent naming really :-) <3